sys/include/net/gnrc/pktbuf.h

```diff
@@ -141,13 +142,26 @@ void gnrc_pktbuf_hold(gnrc_pktsnip_t *pkt, unsigned int num);
 
 /**
  * @brief   Decreases gnrc_pktsnip_t::users of @p pkt atomically and removes it if it
- *          reaches 0.
+ *          reaches 0 and reports a possible error through an error code, if
+ *          @ref net_gnrc_neterr is included.
  *
  * @pre All snips of @p pkt must be in the packet buffer.
  *
  * @param[in] pkt   A packet.
+ * @param[in] err   An error code.
  */
-void gnrc_pktbuf_release(gnrc_pktsnip_t *pkt);
+void gnrc_pktbuf_release_error(gnrc_pktsnip_t *pkt, uint32_t err);
+
+/**
+ * @brief   Decreases gnrc_pktsnip_t::users of @p pkt atomically and removes it if it
+ *          reaches 0 and reports @ref GNRC_NETERR_SUCCESS.
+ *
+ * @param[in] pkt   A packet.
+ */
+static inline void gnrc_pktbuf_release(gnrc_pktsnip_t *pkt)
```

Comment:
To not have to replace every instance of `gnrc_pktbuf_release()` in the code and to not require a user to use `gnrc_pktbuf_release_error()` when there was actually no error.